metaci/testresults/importer.py

```diff
@@ -163,6 +164,24 @@ def import_robot_test_results(build_flow, path):
             )
             classes[result["suite"]["name"]] = testclass
 
+        # Attach suite screenshots to buildflow
+        dirname = os.path.dirname(path)
+        for screenshot in result["suite"]["screenshots"]:
+            if screenshot in suite_screenshots:
+                continue
+            screenshot_path = screenshot
+            if dirname:
+                screenshot_path = "{}/{}".format(dirname, screenshot)
+            with open(screenshot_path, "rb") as f:
+                asset = BuildFlowAsset(
+                    build_flow=build_flow,
+                    asset=ContentFile(f.read(), screenshot),
```

Comment:
This isn't a problem with this PR but I'm surprised that we store photos in the DB instead of on S3.

Reply:
@prescod that's not the case. ContentFile is a thing from django-storages which takes care of sending the file to S3 and holding a reference to it.